ChefExtensionHandler/bin/chef-install.sh

```diff
@@ -91,7 +91,7 @@ chef_install_from_script(){
         exit 1
       elif [ -z "$chef_version" ] && [ -z "$chef_channel" ]; then
         echo "Installing latest chef client"
```

Comment:
Should probably update this message, since it isn't actually the latest version.

Reply:
Thanks @clintoncwolfe & @btm 
I've updated the comment.